src/t-i18n.ts

```diff
@@ -1,132 +1,103 @@
 import { IcuReplacements, Messages, MFunc, SetupOptions, AnyReplacements } from "./types";
-import createCachedFormatter, { CachedFormatter, numberFormatOptions, dateTimeFormatOptions} from "./format";
+import createCachedFormatter, { numberFormats, dateTimeFormats} from "./format";
 import { generator, assign, splitAndEscapeReplacements } from "./helpers";
 import parseIcu from "./icu";
 import parseXml from "./xml";
 
 /**
  * T-i18n - lightweight localization
- * v0.5.0
+ * v0.6.0
  *
  * T-i18n defers to standards to do the hard work of localization. The browser Intl API is use to format
  * dates and numbers. Messages are provided as functions rather than strings, so they can be compiled at build time.
- *
- *
- * Cobbled together by Mitch Cohen on July 31, 2017
```

Comment:
Instead of removing my name, add yours. :)

Reply:
Well it wasn't just your name. This project is no longer a one-day cobbling. :) But I can add names back.